### ISSUE TITLE
Samsung Android WebKit 537.36 does not return engine.name as WebKit

### ISSUE THREAD

**Issue by isaachinman** [faisalman/ua-parser-js#381]
The useragent string below used to return `WebKit` for `engine.name` in v0.7.17, but now does not in v0.7.20.

```
Mozilla/5.0 (Linux; Android 3.0; SM-G900P Build/LRX21T) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.23 Mobile Safari/537.36
```

**Comment by dschwank** [comment]
Same problem here - there seems to be a diff between `0.7.19` and `0.7.20`. `AppleWebKit/537.36` is resolved to `Blink/undefined`.

**Example**
`Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3683.86 Safari/537.36`

**0.7.19**: `"engine": "WebKit/537.36"`
**0.7.20**: `"engine": "Blink/undefined"`

**Comment by dschwank** [comment]
Okay, found this change "causing" the diff:  https://github.com/faisalman/ua-parser-js/commit/3a5a701375ba263a752a2b8df9353fd95f7f9e77

So it's fine for me ... only this regex part is a little odd: `chrome\/(?!27)`

**Comment by mapiram** [comment]
I think it's a fix and not a bug. The Blink engine is featured by the text 'AppleWebKit/537.36' in User-Agent.
Even if I waited the version filled with 357.36 ...

**Comment by drwharris** [comment]
Is this being acknowledged as an issue or not? I've reverted to 0.7.19 because to me its not right.

**Comment by faisalman** [comment]
This is actually a fix for issue #246
